pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryServer.java

```diff
@@ -100,7 +100,7 @@ public void submit(Worker.QueryRequest request, StreamObserver<Worker.QueryRespo
     try {
       distributedStagePlans = QueryPlanSerDeUtils.deserializeStagePlan(request);
     } catch (Exception e) {
-      LOGGER.error("Caught exception while deserializing the request: {}, payload: {}", requestId, request, e);
+      LOGGER.error("Caught exception while deserializing the request: {}", requestId, e);
```

Comment:
this only applies to a single stage of error that occurs during deserializing yes?